### DISCUSSION TITLE
Reduce header/banner size on channel home page

### DISCUSSION THREAD

**Discussion by aindriu80** [ReVanced/revanced-discussions#1050] [General] [answered]
I would like to reduce or completely hide the large banner on the home page that is on every Youtube page without having to build the Revanced app

![reduce_size](https://github.com/user-attachments/assets/6c975280-0dba-466e-8f3c-2b2993f8e571)

The picture above is from the CNN home page, the green box outlines the area I want to reduce or completely remove.

Can this feature be implemented by the Revanced team - I see there is some documentation available but a PR be accepted?

**Comment by oSumAtrIX** [comment]
I don't know what sou mean without building the ReVanced app but this sounds unnecessary to me. Simply scroll down

**Comment by aindriu80** [comment]
when i try to scroll down (using 1/2 of the screen) it doesn't move.  It's very annoying.  I'm using a Samsung Galaxy Tab S9+

I mean patching Revanced with an option for reduced header

**Comment by LisoUseInAIKyrios** [comment, marked as answer]
You can hide most of the channel header with the custom filter:
`^channel_profile.eml`

Not sure if this will incorrectly hide stuff in other places, but it works for the use case you described.

Only the header image remains (above the horizontal scroll links).  Not sure if that can be selectively hidden or not.
